.github/workflows/release.yml

```diff
@@ -286,7 +286,7 @@ jobs:
   resolve-dashboard-image:
     if: github.event.pull_request.draft == false
     needs: goreleaser
-    runs-on: ubuntu-latest
+    runs-on: ${{ vars.DEFAULT_RUNNER }}
```

Comment:
so, this runner is different from `ubuntu-latest-m`, I believe it's a 4 core runner. shall we have another GH var corresponding to that wherever `ubuntu-latest` is used?